firstboot/resize-fs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# resize root partition
+dev=mmcblk0
+total_size="$(/bin/cat /sys/block/$dev/size)"
+last_part="$(cd /dev || exit 1 ; /bin/ls ${dev}p* | /usr/bin/tail -1)"
+last_part_start="$(/bin/cat "/sys/block/$dev/$last_part/start")"
+last_part_size="$(/bin/cat "/sys/block/$dev/$last_part/size")"
+last_part_max="$(("$total_size" - "$last_part_start"))"
+if [ "$last_part_size" -lt "$last_part_max" ] ; then
+	parted "/dev/$dev" resizepart 2 $((total_size-1))s
```

Comment:
```suggestion
	/usr/sbin/parted "/dev/$dev" /usr/bin/resizepart 2 $((total_size-1))s
```
Why ist parted not absolut? All other commands are.

Reply:
Is it really necessary?
The service works the way it is now.. Of course I can modify it, but do we really need it?

Reply:
@naruxde @nbuchwitz 
Thank you